cluster/terraform_kubernetes/config/scripts/hash_password.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# hash_password.sh
```

Comment:
Add `set -eu` to warn the user if there is an error or variable missing

Reply:
fixed